src/DenielWorld/EzTiles/tile/SimpleTile.php

```diff
@@ -58,6 +62,9 @@ public function __construct(Level $level, $tileInfo)
         }
         //Don't mind this, it is for tile recreation after restart which is no longer handled by you.
         if($tileInfo instanceof CompoundTag) {
+
+            $level->loadChunk($this->x >> 4, $this->z >> 4); // fix unloaded chunk at 0 0 issue
```

Comment:
Are you sure this issue only occurs on tile recreations? Or is it possible that it could happen while constructing the tile for the first time?